projects/fsastorefrontlib/src/cms-components/my-account/policy/policies-chart/policies-chart.component.html

```diff
@@ -1,7 +1,7 @@
 <div *ngIf="policiesByPaymentFrequency" class="accordion mb-4">
   <cx-fs-accordion-item
     [opened]="false"
-    [icon]="'fas fa-chart-pie'"
+    [icon]="'fas fas-accordion fa-chart-pie'"
```

Comment:
`fas-accordion` seems redundant if we revert back to previous implementation

Reply:
Because it causes me a problem to position simple inline elements with class fas on the agent locator list view. And that style is applied only on one specific case when the icon is in the accordion title. So I thought it would be nice to have one class for those cases and for others will have fas inline element which is simple to position.

Reply:
Then move `.fs-icon, .fas` from `.accordion -item` to `.accordion-heading`...